STEERING-COMMITTEE.md

```diff
@@ -1,27 +1,108 @@
-5. Manage the Istio brand to decide which things can be called “Istio” and how that mark can be used in relation to other efforts or vendors.
-6. Resolve any dispute from the Technical Oversight Committee.
+The Steering Committee’s responsibilities include:
+
+1. Establishing rules of governance for the Istio project, including creation
+and ratification of bylaws for the Steering Committee and Technical Oversight
+Committee.
+1. Establishing and enforcing principles to guide the Istio project.
+1. Fostering an environment for a healthy and happy community of developers,
+contributors, vendors, and users.
+1. Defining, evolving, and defending a
+[Code of Conduct](CONTRIBUTING.md#code-of-conduct).
+1. Advising the trademark owner on issues relating to the Istio trademark and
+logo.
+1. Setting marketing and advocacy direction for the project; establishing a
+publishing schedule and vetting content, encouraging and assisting in project
+community members, fostering an ecosystem of vendors, creating content for
+conferences, etc.
+1. Controlling and delegating access to, and establishing processes regarding,
+project resources/assets, including but not limited to artifact repositories,
+build and test infrastructure, web sites and their domains, blogs, and social
+media accounts.
+1. Providing neutral mediation as appropriate to try to resolve non-technical
+disputes that arise as part of the project.
+
+## Membership and voting
+1.  The Steering Committee is structured to allow companies who are most
+invested in the success of the Istio project to participate in business and
+non-technical decision-making.
+1.  All members should abide by the project Code of Conduct.
+1.  There are two types of seats on the Steering Committee: Contributor Seats
+and Community Seats.
+1.  Both Contributor and Community Seats will be appointed beginning in
+July 2020. The appointments for the Contribution and Community seat types will
+expire on staggered dates. After the initial term for both seats, all seats
+will have an annual term.
+1.  Contributor Seat terms expire on January 1 and Community Seat terms expire
+on July 1. If necessary, a company holding a Contributor Seat may change the
+appointed individual at any time during the term.
+1.  No Contributing Company can have more than 5 seats in total on the Steering
+Committee.
+    1.  Subsidiaries, an entity in which another company owns more than 50%
+    of the voting or membership interests; and related companies, companies
+    which are controlled by a common third party entity; and affiliate
+    companies, will be treated together as a single entity, referred to as a
+    Contributing Company.
+1.  There shall be nine **Contributor Seats**. Their allocation is determined
+by the approximate effort and expenditure on the Istio project, as approximated
+by the number of merged pull requests on GitHub over the one year period prior
```

Comment:
Question about this metric: at the present time, do PRs from all company contributors get merged at the same speed?  

I know it's normal for reviewers to review their co-worker's PRs first -- this is something we fight against in some of the Kubernetes SIGs -- so I'm a bit concerned that this metric could trigger a kind of feedback loop.

I'm also a bit concerned, based on my Kubernetes experience, that this could trigger a change in how people structure PRs -- encouraging folks to write 3 linked PRs when 1 would have worked.

Reply:
I second these concerns. Ensuring equal PR cycle time is important, and shouldn't be tied in any way to what company you work for.

Reply:
jberkus - Thanks for the comments. These are concerns that all of us have as well. We actually looked at more complex and convoluted strategies, but ultimately decided that the complication might be worse than the value we gain from them. In the end, because *technical* influence (which is what people are most likely to want) resides in the TOC and not in Steering, we decided that it just wasn't likely. But we *did* add the Community seats precisely to allow the community to reward people for non-PR-based contribution.

Still: if you have a better idea, I am absolutely open to it!

Reply:
I am unconcerned with structured PRs. This is a normal part of the review process. Prior to this document publish, I can cite numerous examples where I have both structured PRs, as well as asked developers to structure a PR into multiple for a variety of reasons. If a developer were acting out of malice, I'm not sure why the code of conduct would not apply in these situations, with a continual pattern of what would be considered abuse. The most simple reason to structure a PR - is that it is simpler for the reviewer, and hence merges more quickly...

This is one of the problems with measuring PRs: this charter creates an ethical problem out of something that is not normally one.

Reply:
We will retain "merged PRs" as the metric to set the initial committee and for the January 2021 intake. The charter is now structured to allow the incoming holders of Contributor Seats to change the formula for the next year, if they wish.